packages/react/src/components/TreeView/TreeView.tsx

```diff
@@ -104,6 +179,7 @@ export default function TreeView({
         keys.ArrowDown,
         keys.Home,
         keys.End,
+        // @ts-ignore
```

Comment:
What error is being suppressed here?

Reply:
This is the error I'm suppressing here.
![image](https://github.com/carbon-design-system/carbon/assets/25573926/bb42dd84-c5d9-40c0-9e46-416bbcb6d03d)

Reply:
It's too bad that match.js hasn't been converted to Typescript yet, but I think you can remove the @ts-ignore by just fixing match.js:

```js
/**
 * @typedef Key
 * @property key {Array<string>|string}
 * @property which {number}
 * @property keyCode {number}
 *  @property code {string}        <-- new line here
 */
```